docs/WALKTHROUGH.md

```diff
@@ -80,6 +80,61 @@ export function handleDeposit(event: Deposit): void {
 
 Play around with this, compare hashes on etherscan and help yourself understand some of the behavior.
 
+## Using Logs in Subgraphs
+
+There is a standard logging format that we follow in subgraphs to ensure subgraph data consumers have a framework to understand what is happening in a subgraph. There are five different types of log severity levels that we use in subgraphs:
+
+- `log.debug`: This is only used for debugging purposes. It should not be found in any subgraph code in our `master` branch.
```

Comment:
We can enforce this then at the CI to prevent merges with it

Reply:
I like this https://github.com/messari/subgraphs/issues/1977